modules/microshift-exposed-audit-ports-hostnetwork.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-exposed-audit-ports-hostnetwork_{context}"]
+
+== hostnetwork
+
+When a pod is configured with the `hostNetwork:true` setting, the pod is running in the host network namespace. This configuration can independently open host ports. {microshift-short} component logs cannot be used to track this case, the ports are subject to firewalld rules. If the port opens in firewalld, you can view the port opening in the firewalld debug log.
+//Probably good to like to the firewall docs here, bc it looks like firewall is needed for this 
+
+.Procedure
+
+. Enable debug in the firewalld log by running the following command:
++
+[source,terminal]
+----
+$ sudo vi /etc/sysconfig/firewalld
+FIREWALLD_ARGS=--debug=10
+----
```

Comment:
@pliurh Is it okay to use sudo in this case?

Reply:
Yeah.

Reply:
Awesome, thank you!